catalog/api/app.py

```diff
@@ -0,0 +1,708 @@
+#!/usr/bin/env python3
+
+import asyncio
+import gzip
+import json
+import logging
+import os
+import re
+
+import aiohttp
+from aiohttp import web
+from time import time
+
+import kubernetes_asyncio
+import redis.asyncio as redis
+from hotfix import HotfixKubeApiClient
+from randomstring import random_string
+
+app_api_client = core_v1_api = custom_objects_api = None
+console_url = None
+redis_connection = None
+babylon_namespace = os.environ.get('BABYLON_NAMESPACE')
+interface_name = os.environ.get('INTERFACE_NAME');
+groups = []
+groups_last_update = 0
+admin_api = os.environ.get('ADMIN_API', 'http://babylon-admin.babylon-admin.svc.cluster.local:8080')
+ratings_api = os.environ.get('RATINGS_API', 'http://babylon-ratings.babylon-ratings.svc.cluster.local:8080')
+salesforce_api = os.environ.get('SALESFORCE_API', 'http://salesforce-api.demo-reporting.svc.cluster.local:8080')
+salesforce_authorization_token = os.environ.get('SALESFORCE_AUTHORIZATION_TOKEN')
```

Comment:
I think you are missing  the "salesforce_id_validation" endpoint
@application.route("/api/salesforce/<salesforce_id>", methods=['GET'])